test/api_get_suggestions_test.dart

```diff
@@ -257,7 +260,7 @@ void main() {
     test('Suggestions for additives', () async {
       List<String> result = await OpenFoodAPIClient.getSuggestions(
         TagType.ADDITIVES,
-        language: OpenFoodFactsLanguage.RUSSIAN,
```

Comment:
What impact does this have

Reply:
For some reason the test does not pass anymore in Russian.
As we more or less agreed that it was more relevant to run integration tests with server data than unit tests with local json files, once in a while we have to "fix" tests.